### PR TITLE
Small QOL updates

### PR DESCRIPTION
**Pull request by NinjaLabs-Dev**
- Stopped mouse selecting text on about me items
- Added transition on social icon hover

### CODE REVIEW

**Comment by NinjaLabs-Dev** [comment]
Any update on this?

**Comment by ProgrammerGaurav** [comment]
> Any update on this?

I have added hacktoberfest-accepted label to your PR so it is eligible for fest,  and I need to review it completely before merging....so it will take some time.....

**Comment by NinjaLabs-Dev** [comment]
@ProgrammerGaurav No worries! 😄

File: css/style.css [hunk 1]
```diff
@@ -4,8 +4,7 @@ body {
   margin: 0;
   padding: 0;
   color: #ffffff;
-  font-family: 'Ubuntu', sans-serif;
-}
+  font-family: 'Ubuntu', sans-serif; }
```
Comment (ProgrammerGaurav) [inline css/style.css:7]: Most of the CSS changes are because of the Last braces!

File: css/style.css [hunk 2]
```diff
@@ -15,50 +14,35 @@ body {
...
-  border-radius: 10px;
-  padding: 15px;
-}
-
-.navbar .navbar-nav .nav-link:hover {
-  background-color: #ffffff;
-  color: #000000;
-}
+  transition-duration: 0.5s; }
+  .navbar .navbar-brand {
```
Comment (ProgrammerGaurav) [inline css/style.css:31]: CSS is not formatted, and not an imp change

File: css/style.css [hunk 3]
```diff
@@ -93,27 +73,22 @@ div.type-wrap {
   font-weight: bold;
   top: 50%;
   left: 50%;
-  -webkit-transform: translate(-50%, -50%);
```
Comment (ProgrammerGaurav) [inline css/style.css:96]: WebKit is important deleting can cause problem

File: css/style.css [hunk 4]
```diff
@@ -126,56 +101,41 @@ div.type-wrap {
   display: block;
   position: relative;
   overflow: hidden;
-  -webkit-transition: all 0.3s;
   transition: all 0.3s;
-  -webkit-box-shadow: 0 0 50px 5px #b1b1b1;
```
Comment (ProgrammerGaurav) [inline css/style.css:131]: Webkit deleted

File: css/style.css [hunk 5]
```diff
@@ -126,56 +101,41 @@ div.type-wrap {
...
-  font-size: 50px;
-}
-
-.skills img {
-  width: 100px;
-  height: 100px;
-  padding-right: 20px;
-  padding-left: 20px;
-}
+  text-align: center; }
```
Comment (ProgrammerGaurav) [inline css/style.css:116]: Same Code no change!

File: css/style.css [hunk 6]
```diff
@@ -126,56 +101,41 @@ div.type-wrap {
...
-
-.certifications h2 {
-  margin-top: 50px;
-}
-
-.certifications img {
-  height: 254px;
-  padding: 10px;
-}
+  text-align: center; }
```
Comment (ProgrammerGaurav) [inline css/style.css:133]: Same Code no change!

File: css/style.css [hunk 7]
```diff
@@ -185,74 +145,50 @@ div.type-wrap {
...
-.footer .fa-github {
-  background: #212529;
-  color: white;
-}
-
-.footer .soc {
-  display: inline-block;
-  margin: 4px;
-}
+  background-size: cover; }
```
Comment (ProgrammerGaurav) [inline css/style.css:148]: Same Code no change!

File: css/style.css [hunk 8]
```diff
@@ -185,74 +145,50 @@ div.type-wrap {
...
-  width: 100%;
-  background-color: #333333;
-  color: #fff;
-}
-
-.contact button:hover {
-  color: #fff;
-  background-color: #444444;
-}
+  background-color: #FAFAFA; }
```
Comment (ProgrammerGaurav) [inline css/style.css:186]: Same Code no change!

File: css/style.scss [hunk 9]
```diff
@@ -131,6 +131,10 @@ div.type-wrap {
     overflow: hidden;
     transition: all 0.3s;
     box-shadow: 0 0 50px 5px rgb(177, 177, 177);
+
+    p {
+        user-select: none;
```
Comment (ProgrammerGaurav) [inline css/style.scss:136]: This is great, and the only good thing in the code

**Review by ProgrammerGaurav — COMMENTED** [review]
The Code is good but not good enough for merging to the master branch

**Comment by NinjaLabs-Dev** [comment]
@ProgrammerGaurav All of these issues are caused by my scss compiler. Which one did you use so I can replicate the original code?

**Pull request closed without merging**